sklearn/decomposition/tests/test_kernel_pca.py

```diff
@@ -282,3 +283,15 @@ def test_kernel_conditioning():
     # check that the small non-zero eigenvalue was correctly set to zero
     assert kpca.lambdas_.min() == 0
     assert np.all(kpca.lambdas_ == _check_psd_eigenvalues(kpca.lambdas_))
+
+
+@pytest.mark.parametrize("kernel",
+                         ["linear", "poly", "rbf", "sigmoid", "cosine"])
+def test_kernel_pca_inverse_transform(kernel):
+    X, *_ = make_blobs(n_samples=100, n_features=4, centers=[[1, 1, 1, 1]],
+                       random_state=0)
+
+    kp = KernelPCA(n_components=2, kernel=kernel, fit_inverse_transform=True)
+    X_trans = kp.fit_transform(X)
+    X_inv = kp.inverse_transform(X_trans)
+    assert np.isclose(X, X_inv).all()
```

Comment:
you can use `assert_allclose`: `from sklearn.utils._testing import assert_allclose`

```suggestion
    assert_allclose(X, X_inv)
```